plugin/federation/federation.gotpl

```diff
@@ -156,14 +156,12 @@ func (ec *executionContext) __resolve_entities(ctx context.Context, representati
 						}
 
 						for i, entity := range entities {
-							{{- range $entity.Requires -}}
-								{{- range .Fields -}}
```

Comment:
So won't removing this break cases with multiple fields?

Reply:
`Requires` doesn't have `Fields` field.
I have added [tests](https://github.com/chedom/gqlgen/commit/bff3236d8c1842aa2f6ef8bfc874dea07d9b9cd8#diff-edfaf4289eb8d3b4c9284207d6310570a74c3d2d31f5c1bbb6d4e721a33edc4a) for Entity with multiple required fields (@requires(fields: "key1 key2"). It works as expected.